libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.EndDialog.schema

```diff
@@ -1,15 +1,17 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "End Dialog",
-    "description": "Command which ends the current dialog, returning the resultProperty as the result of the dialog.",
+    "title": "End dialog",
+    "description": "End this dialog.",
     "type": "object",
     "properties": {
         "value": {
             "$role": "expression",
-            "description": "Specifies a value expression for the result to be returned to the caller.",
+            "title": "Value",
+            "description": "Value that will be passed back to the parent dialog.",
```

Comment:
Value that will be passed back to the parent dialog as the result for this dialog.